src/utils/index.ts

```diff
@@ -7,4 +8,21 @@ export function isValidPrivateKey(privateKey: string) {
     } catch (error) {
         return false;
     }
+}
+
+export async function promptUserKeyCustomizationDecision () : Promise<boolean> {
+    return await select({
+        message:
+          "Would you like to use test configuration keys for testing, or customize your configuration keys?",
```

Comment:
Let's change slightly to "Would you like to use default configuration keys for testing..."

Reply:
Done. Modified to: "Would you like to use the default configuration keys for testing?"